doc/man/mc.1.in

```diff
@@ -79,7 +79,7 @@ script that automatically changes the current directory of the shell to
 the last directory Midnight Commander was in. Source the file
 .B %pkglibexecdir%/mc.sh
 (bash and zsh users) or
-.B %libexecdir%/mc.csh
+.B %pkglibexecdir%/mc.csh
```

Comment:
actually there's another issue: `pkglibexecdir` is not defined on my system, only `libexecdir` is.
After installing, `man mc` renders like:

```
              shell to the last directory Midnight Commander was in. Source the file @pkglibexecdir@/mc.sh (bash and zsh
              users) or /home/johannes/.local/libexec/mc.csh (tcsh users) respectively to define mc as an alias  to  the
```

I'm building with

```
./autogen.sh &&
    ./configure --prefix=$HOME/.local &&
    make -j3 &&
    make install
```